hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/routes.tsx

```diff
@@ -23,7 +23,8 @@ import {NotFound} from './views/notFound/notFound';
 import {IRoute} from './types/routes.types';
 import {MissingContainers} from './views/missingContainers/missingContainers';
 import {Insights} from './views/insights/insights';
-import {DiskUsage} from './views/diskUsage/diskUsage';
+import { DiskUsage } from './views/diskUsage/diskUsage';
+import {HeatMap} from './views/heatMap/heatMap';
```

Comment:
if you want to do the rename in this PR
```suggestion
import {Heatmap} from './views/heatmap/heatmap';
```